src/browser/modules/D3Visualization/lib/visualization/components/layout.ts

```diff
@@ -40,20 +40,7 @@ type ForceLayout = {
 export type Layout = { init: (render: () => void) => ForceLayout }
 export type AvailableLayouts = Record<'force', () => Layout>
 
-let simulationTimeout: null | number = null
-export const setSimulationTimeout = (
-  simulation: Simulation<VizNode, Relationship>
-) => {
-  simulationTimeout = setTimeout(() => simulation.stop(), 2000)
-}
-export const clearSimulationTimeout = () => {
-  if (simulationTimeout) {
-    clearTimeout(simulationTimeout)
-    simulationTimeout = null
-  }
-}
-
-const SIMULATION_STARTING_TICKS = 800
+const SIMULATION_STARTING_TICKS = 300
```

Comment:
Because the frames still take a little while to show up, it's easy to run into this issue:

https://user-images.githubusercontent.com/10564538/150496195-bad6113d-8cdb-401c-b501-a4c55fa83c61.mp4

I don't think changing the starting ticks more would solve the problem and I don't think it's something you'd need to look into for the d3 changes though.